src/pages/index.tsx

```diff
@@ -201,7 +212,18 @@ class IndexPage extends Component<IndexPageProps, IndexPageState> {
 						renderTableRow={crew => this.renderTableRow(crew)}
 						filterRow={(crew, filter) => this._filterCrew(crew, filter)}
 						config={tableConfig}
-					/>
+                        searchExt = { 
+                            <span style={{ paddingLeft: '2em' }}>
+                                <Dropdown inline
+                                          options={searchTypeOptions}
```

Comment:
```suggestion
                                          selection
                                          options={searchTypeOptions}
```

This addresses @flash82's comment about styling consistency.